index.html

```diff
@@ -69,10 +69,10 @@ <h1 class="intro-text">I’m Azeem
 
     <div class="work">
       <div class="right-block">
-        <img src="image\SnapshootPortfolio.png" alt=" " />
+        <img src="image\SnapshootPortfolio.png" alt="portfolio image 1 " />
       </div>
       <div class="desktopad1">
-        <img src="image\desktopad1.png" alt="" />
+        <img src="image\desktopad1.png" alt="portfolio image" />
       </div>
       <div class="left-block">
         <h3 class="work-title">Tonic</h3>
```

Comment:
- [ ] You have some contrast ratio ("color contrast") on each technology used in a project, kindly make sure it has the proper ratio score.
![image](https://user-images.githubusercontent.com/33767951/139862512-ad4b0a08-4e13-4808-a52d-2beba087a1a4.png)